.circleci/ci/src/jobs/job-publish-pr-env-urls.ts

```diff
@@ -0,0 +1,60 @@
+import { commands, Config, Job, reusable } from '@circleci/circleci-config-sdk';
+import { NodeLtsExecutor } from '../executors';
+import { orbs } from '../orbs';
+import { config } from '../config';
+import { NotifyOnFailureCommand } from '../commands';
+
+export class PublishPrEnvUrlsJob {
+  private static jobName = 'job-publish-pr-env-urls';
+
+  public static create(dynamicConfig: Config): Job {
+    dynamicConfig.importOrb(orbs.keeper);
+    dynamicConfig.importOrb(orbs.github);
+
+    const notifyOnFailureCommand = NotifyOnFailureCommand.get(dynamicConfig);
+    dynamicConfig.addReusableCommand(notifyOnFailureCommand);
+
+    return new Job(PublishPrEnvUrlsJob.jobName, NodeLtsExecutor.create('small'), [
+      new commands.Checkout(),
```

Comment:
I'm wondering if we really need to checkout

Reply:
done